### ISSUE TITLE
Bad state in ancestoral sequence

### ISSUE THREAD

**Issue by lskatz** [rambaut/Seq-Gen#25]
Can anyone help me understand this error? I can run `-k` with the example sequences from the repo, but not with my own large phylip/tree file.

```bash
$ seq-gen -m GTR -p 1 -n 1 -s 1 -k 1 seqs+trees.phy
Sequence Generator - seq-gen
Version 1.3.4
(c) Copyright, 1996-2017 Andrew Rambaut and Nick Grassly
Institute of Evolutionary Biology, University of Edinburgh

Originally developed at:
Department of Zoology, University of Oxford

Random number generator seed: -8885896833777932125

Simulations of 4 taxa, 40 nucleotides
  for 1 tree(s) with 1 dataset(s) per tree

Alignment read: numSequences = 4, numAlignmentSites = 40
Using sequence 1 as the ancestral sequence

Branch lengths of trees multiplied by 1

Rate homogeneity of sites.
Model = GTR: General time reversible (nucleotides)
  Rate of transitions and transversions equal:
  rate matrix = gamma1: 1.0000 alpha1: 1.0000  beta1: 1.0000
                                beta2: 1.0000 alpha2: 1.0000
                                              gamma2:  1.0000
  with nucleotide frequencies equal.
 4 40
Taxon1    ATAGTTGTAGTCATCCTGATATCACTATTCTTAGATCTTA
Taxon2    ACATTTCTATGCATCCTCATATTACCATTCTTCGATCTAA
Taxon3    ATCGTTGTAGGAATTCGCGGATTAGCACTCTCGAATCTAA
Taxon4    ATCTTGCTAGTAATCGGTGTGTTTGCAATCGTATAGCAAA
Time taken: 0 seconds
```

```bash
$ seq-gen -m GTR -p 1 -n 1 -s 1 -k 1 /scratch/make-models-hYtpwl/phylip.tree
Sequence Generator - seq-gen
Version 1.3.4
(c) Copyright, 1996-2017 Andrew Rambaut and Nick Grassly
Institute of Evolutionary Biology, University of Edinburgh

Originally developed at:
Department of Zoology, University of Oxford

Random number generator seed: -5257390016733364268

Simulations of 1602 taxa, 1392 nucleotides
  for 1 tree(s) with 1 dataset(s) per tree

Alignment read: numSequences = 1602, numAlignmentSites = 1392
Using sequence 1 as the ancestral sequence

Branch lengths of trees multiplied by 1

Rate homogeneity of sites.
Model = GTR: General time reversible (nucleotides)
  Rate of transitions and transversions equal:
  rate matrix = gamma1: 1.0000 alpha1: 1.0000  beta1: 1.0000
                                beta2: 1.0000 alpha2: 1.0000
                                              gamma2:  1.0000
  with nucleotide frequencies equal.
Bad state in ancestoral sequence
```

**Comment by lskatz** [comment]
I had more than 10 characters in my taxon names....

**Comment by lskatz** [comment]
I have found other issues that give me the same error `Bad state in ancestoral sequence`

* I had to change nucleotides to uppercase
* Word wrapping broke it and so I used `goalign reformat --phylip phylip --one-line`
* It worked if I reduced the input phylip sequence length. In one case, I had to change the header from `    2344 1974` to `    2344 1199`. `1200` did not work but lower numbers did. This maximum number changes per phylip file and so I cannot figure out what the pattern is.
